R/class-RDAPlot.R

```diff
@@ -114,8 +135,8 @@ setMethod(".createObservers", "RDAPlot",
     panel_name <- .getEncodedName(x)
     
     .createProtectedParameterObservers(panel_name, c("dimred", "add.ellipse",
-        "colour_by", "vec.text", "add.vectors"), input=input, pObjects=pObjects,
-        rObjects=rObjects)
+        "colour_by", "vec.text", "add.vectors", "add.expl.var", "add.significance"),
```

Comment:
Probably unresponsive to confidence.level, because you need to add it to the observers here